source/AI.cpp

```diff
@@ -2595,6 +2600,22 @@ void AI::DoScatter(Ship &ship, Command &command)
 
 
 
+// Rove; i. e. wander aimlessly around the system
+void AI::Rove(Ship &ship, Command &command) 
+{
+	// If this is a carried ship (like a fighter), or if the ship is disabled or destroyed, don't rove.
+	if(ship.CanBeCarried() || ship.IsDisabled() || ship.IsDestroyed())
+		return;
+
+	// Create a random point and move to it.
+	int areaSize = 10000;
+	Point destination(Random::Real()*areaSize-areaSize/2, Random::Real()*areaSize-areaSize/2);
+	Point velocity(0, 0);
+	MoveTo(ship, command, destination, velocity, ship.MaxVelocity(), ship.MaxVelocity());
```

Comment:
Points are created as (0,0), so no need to specify it. Also little need to create a variable you never actually use.
```suggestion
	MoveTo(ship, command, destination, Point(), ship.MaxVelocity(), ship.MaxVelocity());
```